.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,150 @@
+name: Node.js CI
+on:
+  workflow_call:
+    inputs:
+      python_v:
```

Comment:
I think we could remove this , it makes sense for the python repos because we have python 2 and python 3 code... but here it's only for pre-commit

Reply:
done